vic/drivers/classic/src/vic_classic.c

```diff
@@ -109,17 +109,17 @@ main(int   argc,
     // Check that model parameters are valid
     validate_parameters();
 
+    /** Make Date Data Structure **/
+    initialize_time();
+    dmy = make_dmy(&global_param);
+
     /** Set up output data structures **/
-    out_data = create_output_list();
+    set_output_met_data_info();
+    out_data = create_outdata(1);
+    fclose(filep.globalparam);
     filep.globalparam = open_file(filenames.global, "r");
-    parse_output_info(filep.globalparam, &out_data_files, out_data);
-    for (filenum = 0; filenum < options.Noutfiles; filenum++) {
-        if (out_data_files[filenum].nvars == 0) {
-            log_err("No output variables were set in OUTFILE %zu. "
-                    "Must set at least one output variable (OUTVAR) "
-                    "for each OUTFILE.", filenum + 1);
-        }
-    }
+    parse_output_info(filep.globalparam, &streams, &(dmy[0]));
+    // validate_streams(&streams);
```

Comment:
finish this function.  Things to check:
- `nvars > 0`, `ngridcells > 0`
- `agg_alarm` is set and that the agg step fits within expected bounds
- `prefix` is set
- `file_format` and `compress` are set.
- all arrays have been allocated (`type`, `mult`, `varid`, `aggtype`, `aggdata`)